### PR TITLE
Fixed constants TS typings

### PR DESCRIPTION
**Pull request by JuanM04**
Re-added `src/constants/index.d.ts` and added module definitions to `typings.d.ts`

### CODE REVIEW

File: scripts/build/_lib/typings/typeScript.js [hunk 1]
```diff
@@ -192,6 +192,23 @@ function getTypeScriptLocaleIndexModuleDefinition(submodule, locales) {
   }
 }
 
+function getTypeScripttConstantsModuleDefinition(constants, fnSuffix) {
```
Comment (fturmel) [inline scripts/build/_lib/typings/typeScript.js:195]: Little typo here, two `t`s in TypeScript

**Review by fturmel — CHANGES_REQUESTED** [review]
Nice! I also had noticed that the constants `.d.ts` file wasn't automatically generated.

File: scripts/build/_lib/typings/typeScript.js (hunk 1 above)
Comment (JuanM04) [reply scripts/build/_lib/typings/typeScript.js]: Nice catch, thanks

**Review by fturmel — APPROVED** [review]

**Review by leshakoss — APPROVED** [review]
Thank you so much!

**Pull request merged**